### ISSUE TITLE
Space gets wet and runtimes

### ISSUE THREAD

**Issue by skull132** [Aurorastation/Aurora.3#1026]
```
runtime error:
[17:23:06]undefined variable /turf/space/var/wet
[17:23:06]proc name: wet floor (/turf/simulated/proc/wet_floor)
[17:23:06]  source file: simulated.dm,25
```

**Comment by NanakoAC** [comment]
should be fixed by #946 
I defined the wet floor proc at the /turf level with an empty return, and overrode it at turf/simulated

**Comment by skull132** [comment]
Checked new iterations of code, everything looks properly checked against. So report is outdated until further notice.
